news/_posts/2015-06-09-GSoC2015:Week_1_and_2_Updates.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: " GSoC2015: Weeks 1 and 2"
+authors: Amit
+date: "2015-06-09 11:36"
+---
+
+During the past two weeks we have worked on the following :
+
+  a. Common `QuPropagator` type which has the numerical method as one of the constructors.
+  b. Common `QuPropagatorState` which has the current state, the current time and the current time state (the next index from time step list)
+  c. `QuPropagatorMethod` parent of all the numerical methods.
+  d. Docile integration
+  e. Tests
+
+The idea of `QuPropagator` is to unify the creation of parameters involved in solving the differential equation. Its creation also involves passing of the numerical method used to solve the problem. So for example:
+
+```
+Hamiltonian = sigmax
+Initial state(qv) = normalize!(QuArray([0.5+0.1im, 0.2+0.2im]))
```

Comment:
I would make this Julia syntax, i.e.
```
# initial state
qv = ...
# time steps
t = ...
```

Note that you don't need the parentheses `[...]` for the latter, a Range will also work and is (probably) more efficient.